src/pages/StaticPage.tsx

```diff
@@ -1,3 +1,27 @@
+import {  useNavigate } from "react-router-dom"
+// import { icons } from '../images/<file>'
+
 export default function StaticPage({ title }: { title: string }) {
```

Comment:
Lets create an interface for this, and have the component accept an icon as a property.